packages/babel-parser/test/helpers/run-fixture-tests.js

```diff
@@ -7,9 +7,23 @@ import toFuzzedOptions from "./to-fuzzed-options.js";
 import { serialize, deserialize } from "./serialization.js";
 import toContextualSyntaxError from "./to-contextual-syntax-error.js";
 
-const { OVERWRITE } = process.env;
+const { CI, OVERWRITE } = process.env;
 const { stringify, parse: JSONParse } = JSON;
 
+/**
+ * run parser on given tests
+ *
+ * @param {Test} A {@link packages/babel-helper-fixtures/src/index.js Test}
```

Comment:
The param should be a `string`.

Reply:
Oops! Fixed, and updated explanation too: https://github.com/tolmasky/babel/commit/9b87eeaeb5ba81d228c92e8a34f236f777d96226